suites/message/puppet.go

```diff
@@ -0,0 +1,40 @@
+package message
+
+import (
+	"context"
+	"testing"
+
+	"github.com/filecoin-project/chain-validation/drivers"
+	"github.com/filecoin-project/chain-validation/state"
+	abi_spec "github.com/filecoin-project/specs-actors/actors/abi"
+	big_spec "github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/filecoin-project/specs-actors/actors/puppet"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+)
+
+func MessageTest_PuppetActorInnerSends(t *testing.T, factories state.Factories) {
+	builder := drivers.NewBuilder(context.Background(), factories).
+		WithDefaultGasLimit(1_000_000).
+		WithDefaultGasPrice(big_spec.NewInt(1)).
+		WithActorState(append(drivers.DefaultBuiltinActorsState, drivers.PuppetActorState))
```

Comment:
This would be more ergonomic if `WithActorState` _added_ an actor to the builtin set rather than replacing the builtin set (which requires the caller to do an `append`, which they will always do).

I don't quite understand how this works without requiring the puppet actor's address as a parameter specifying where to put the state. But something like:
```
	.WithActor(myPuppetAddr, myPuppetCodeID, puppet.ConstructState(), myPuppetBalance)
	.WithActor(anotherAddr, someCodeId, foo.ConstructState(), anotherBalance)
```
would make sense.